Hymns/Tags/TagEntity.swift

```diff
@@ -6,17 +6,23 @@ class TagEntity: Object, Identifiable {
     @objc dynamic var hymnIdentifierEntity: HymnIdentifierEntity!
     @objc dynamic var songTitle: String!
     @objc dynamic var tag: String!
+    @objc private dynamic var privateTagColor: Int = TagColor.blue.rawValue
```

Comment:
Can you add `// https://stackoverflow.com/questions/29123245/using-enum-as-property-of-realm-model` as a comment on this field

Reply:
done